Gemfile

```diff
@@ -1,47 +1,41 @@
 source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "2.3.0"
+ruby '2.7.1'
```

Comment:
Travis can't run if the gemfile says 2.7.1 but the travis.yml says 2.4

Reply:
good point! I updated Travis's configuration 👍